src/api/middleware/validators/company.js

```diff
@@ -77,3 +77,19 @@ export const getOffers = useExpressValidators([
         .custom(isObjectId).withMessage(ValidationReasons.OBJECT_ID).bail()
         .custom(companyExists).withMessage(ValidationReasons.COMPANY_NOT_FOUND)
 ]);
+
+export const checkConcurrent = useExpressValidators([
+    existingCompanyParamValidator,
+
+    body("publishDate", ValidationReasons.DEFAULT)
+        .exists().withMessage(ValidationReasons.REQUIRED).bail()
+        .isISO8601({ strict: true }).withMessage(ValidationReasons.DATE).bail()
+        .customSanitizer(normalizeDate),
+
+    body("publishEndDate", ValidationReasons.DEFAULT)
+        .exists().withMessage(ValidationReasons.REQUIRED).bail()
```

Comment:
I think that if `publisEndhDate` is not passed it should default to `publishDate + max_offer_duration`, not being required in the request. If none are passed, `publishEndDate` would be `now +  max_offer_duration`